docs/access-handler/runtimes.md

```diff
@@ -0,0 +1,15 @@
+## Runtimes
+
+The access handler has support for different runtimes, this may allow the access handle to run in different Clouds eventually.
+For now, we have a local runtime and a lambda runtime.
```

Comment:
```suggestion
The access handler has support for different runtimes, this may allow the access handle to run in different clouds eventually. For now, we have a local runtime and a lambda runtime.
```